node/validator_store.go

```diff
@@ -128,211 +80,97 @@ type validatorStore struct {
 var _ ValidatorStore = &validatorStore{}
 
 func NewValidatorStore(
-	ctx context.Context,
 	logger logging.Logger,
 	config *Config,
 	timeSource func() time.Time,
 	ttl time.Duration,
 	registry *prometheus.Registry) (ValidatorStore, error) {
 
-	if !config.LittDBEnabled {
-		logger.Warn("WARNING: This node is running with littDB disabled. " +
-			"This is a deprecated mode of operation, and will not be supported in future versions.")
+	if len(config.LittDBStoragePaths) == 0 {
+		return nil, fmt.Errorf("no littDB paths provided")
 	}
 
-	littDBPath := path.Join(config.DbPath, LittDBPath)
-	levelDBPath := path.Join(config.DbPath, LevelDBPath)
-	levelDBDeletionPath := path.Join(config.DbPath, LevelDBDeletionPath)
-
-	// If we previously made an attempt at deleting the levelDB database but it was interrupted, delete it now.
-	exists, err := util.Exists(levelDBDeletionPath)
-	if err != nil {
-		return nil, fmt.Errorf("failed to stat path %s: %v", levelDBDeletionPath, err)
+	stringBuilder := strings.Builder{}
+	if len(config.LittDBStoragePaths) > 1 {
+		stringBuilder.WriteString("s")
```

Comment:
this is confusing when reading this function linearly, consider writing `Using littDB at path` to the string builder, and then just doing an `logger.Info(stringBuilder.String())` below

Reply:
makes sense, done